### PR TITLE
Common Regions Geojson

### PR DESCRIPTION
**Pull request by deepak-shah-np**
The common regions geojson file is generated from  countries geojson(shapefile) which  is sourced from the [Natural Earth's official repository, specifically version 5.1.1](https://www.naturalearthdata.com/downloads/50m-cultural-vectors/50m-admin-0-countries-2/).

The common regions are r5, r9 and r10 which is based on [common regions definitions](https://github.com/IAMconsortium/common-definitions/blob/main/definitions/region/common.yaml)

### CODE REVIEW

File: geojson/readme.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+The initial countries geojson(shapefile) [world_ne](world_ne.geojson) is sourced from the [Natural Earth's official repository, specifically version 5.1.1](https://www.naturalearthdata.com/downloads/50m-cultural-vectors/50m-admin-0-countries-2/).
```
Comment (danielhuppmann) [inline geojson/readme.md:1]:
```suggestion
The initial countries geojson (shapefile) [world_ne](world_ne.geojson) is sourced from the [Natural Earth's official repository, version 5.1.1](https://www.naturalearthdata.com/downloads/50m-cultural-vectors/50m-admin-0-countries-2/).
```

Comment (danielhuppmann) [inline geojson/readme.md:1]: Please add a title to the readme

File: geojson/readme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## Common Regions
+
+The geojson file for common regions is generated according to the [IAMC Common Definitions](https://github.com/IAMconsortium/common-definitions/blob/main/definitions/region/common.yaml).
+
+- r5 regions
+
+  OECD & EU (R5), Reforming Economies (R5), Asia (R5),Middle East & Africa (R5),Latin America (R5)
```
Comment (danielhuppmann) [inline geojson/readme.md:9]:
```suggestion
  OECD & EU (R5), Reforming Economies (R5), Asia (R5), Middle East & Africa (R5), Latin America (R5)
```

File: geojson/readme.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- r9 regions
+
+  European Union (R9), USA (R9), Other OECD (R9), China (R9), India (R9),Other Asia (R9), Reforming Economies (R9), Middle East & Africa (R9), Latin America (R9)
```
Comment (danielhuppmann) [inline geojson/readme.md:13]:
```suggestion
  European Union (R9), USA (R9), Other OECD (R9), China (R9), India (R9), Other Asia (R9), Reforming Economies (R9), Middle East & Africa (R9), Latin America (R9)
```

**Review by danielhuppmann — COMMENTED** [review]
Thanks @deepak-shah-np - just one minor request, can you please rename the readme to `README.md`, add my clean-up suggestions, and add line breaks after 80 characters.

File: geojson/readme.md (hunk 1 above)
Comment (deepak-shah-np) [reply geojson/readme.md]: Done!

Comment (deepak-shah-np) [reply geojson/readme.md]: Done!

File: geojson/readme.md (hunk 2 above)
Comment (deepak-shah-np) [reply geojson/readme.md]: Done!

File: geojson/readme.md (hunk 3 above)
Comment (deepak-shah-np) [reply geojson/readme.md]: Done!

**Review by danielhuppmann — APPROVED** [review]
Thanks @deepak-shah-np!

**Pull request merged**